<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nosic</class>
 <widget class="QWidget" name="nosic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(43, 43, 43);
border-top:2px solid #2b2b2b;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="jpg_label">
      <property name="geometry">
       <rect>
        <x>158</x>
        <y>174</y>
        <width>300</width>
        <height>302</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>302</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>302</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">min-width:     300px;    
min-height:    300px;     
max-width:     300px;    
max-height:    300px;     
border-radius: 150px;      </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/new/prefix2/music/Jpg/00.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>109</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(137, 138, 139);
font: 75 10pt &quot;阿里妈妈数黑体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(137, 138, 139);
font: 75 10pt &quot;阿里妈妈数黑体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>262</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(191, 191, 191);
font: 75 14pt &quot;阿里妈妈数黑体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>661</x>
        <y>325</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(191, 191, 191);font: 75 14pt &quot;阿里妈妈数黑体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>447</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(137, 138, 139);
font: 75 10pt &quot;阿里妈妈数黑体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>496</y>
        <width>400</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(137, 138, 139);
font: 75 10pt &quot;阿里妈妈数黑体&quot;;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/new/prefix2/wyy res/QQ2023070119263222.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_1</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>label</zorder>
     <zorder>jpg_label</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
